laravel-app/resources/views/search/results.blade.php

```diff
@@ -20,11 +20,26 @@
 									<div class="col text-start">
 										<a class="text-dark" href="{{ route('profile.show', $result->id) }}">
 											<img src="{{ asset('assets/images/microblog-logo-iconx30.png') }}" alt="Image">
-											{{ $result->username }}
+											{{ $result->username }} {{$result->id}}
 										</a>
 									</div>
-									<div class="col-2">
+									<!-- <div class="col-2">
 										<a href="#" class="btn btn-dark">Follow</a>
+									</div> -->
+									<div class="col-3 align-self-center">
+										@if(auth()->user()->isNot($result))
```

Comment:
too much nesting

Reply:
done sir, used the method isFollow to avoid nesting. Thank you ❤️